bindings/java/Makefile

```diff
@@ -24,6 +24,7 @@ build: gradlew
 	mkdir -p $(OUT_DIR)
 	javac ./java/src/main/java/org/ethereum/evmc/EvmcVm.java -h ./c --class-path ./java/src/main/java -s ./java/build
 	mv c/org_ethereum_evmc_EvmcVm.h c/evmc-vm.h
+	clang-format -i c/evmc-vm.c c/host.c c/host.h
```

Comment:
Don't want to nitpick, but this probably doesn't belong here? (It probably simplifies your submission process though.)

Reply:
no worries.  it broke my build 😂.  already moved it.